buildpipeline/Core-Setup-CrossBuild.json

```diff
@@ -286,6 +288,15 @@
     "SourceVersion": {
       "value": "",
       "allowOverride": true
+    },
+    "DockerHost_Sandbox": {
+      "value": "$(Build.StagingDirectory)/HostSandbox"
```

Comment:
I'm conflicted on this, vs. `Build.BinariesDirectory` (and enabling cleanup of that dir). The doc has a pretty specific purpose for `Build.StagingDirectory`, but `Binaries` is pretty specific, too. I'm fine leaving it Staging, but I wish there was a clear fit.